app/errors.py

```diff
@@ -0,0 +1,31 @@
+"""
+This module holds possible errors
+"""
+
+
+class NotWearingMaskError(Exception):
+    """
+    No mask -> virus gets in -> die

```

Comment:
I got the joke :)
But code should be readable and clear, so it's not a good idea to write jokes in the docs. It's better to write descriptive and informative messages.
You can share your jokes in the chats, it would be better :)

Could you rewrite or remove such docs, please?)
Btw, it's nice that you decided to write docstrings, go on!